CDTDatastore/HTTP/CDTURLSession.m

```diff
@@ -33,6 +34,7 @@ @implementation CDTURLSession
 
 - (instancetype)init
 {
+    self.asyncTaskMonitor = dispatch_semaphore_create(4);
```

Comment:
This line is redundant as it gets set in `initWithCallbackThread:requestInterceptors:sessionConfigDelegate:`